### PR TITLE
Improve styling of headers used in the pullquote

### PR DESCRIPTION
**Pull request by Axinet**
Hello,
As you probably noticed, it is possible to use your plugin with headers placed in the shortcode content. 
This pull request improves the styling of such usage from stylesheet perspective.

_To be honest, there is an HTML quirk in the outputted code when using headers, but I need more time to get rid of it right (and it already exists and doesn't block headers usage)._

Before this PR:

With this PR:
prcss

### CODE REVIEW

**Comment by adamdehaven** [comment]
To be honest, the plugin isn't designed for placing headers within the pullquote. I realize it's possible to do so, but considering there are infinite ways someone's theme can style header elements, trying to handle headers within the pullquote is most-likely outside the scope of the plugin.

TLDR: I assume users won't place headers inside of the shortcode.

If you want to keep going on this track, I'll take a look at the pull request when you're ready.
